pkg/server/helpers_windows.go

```diff
@@ -19,10 +19,39 @@ limitations under the License.
 package server
 
 import (
+	runhcsoptions "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/options"
+	criconfig "github.com/containerd/cri/pkg/config"
+	"github.com/pkg/errors"
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
 // initSelinuxOpts is not supported on Windows.
 func initSelinuxOpts(selinuxOpt *runtime.SELinuxOption) (string, string, error) {
 	return "", "", nil
 }
+
+func (c *criService) getSandboxPlatform(sandboxID string) (string, error) {
+	sandbox, err := c.sandboxStore.Get(sandboxID)
+	if err != nil {
+		return "", err
+	}
+	var sandboxPlatform string
+
+	// Get the RuntimeHandler config overrides
+	var ociRuntime criconfig.Runtime
+	if sandbox.RuntimeHandler != "" {
+		ociRuntime = c.config.Runtimes[sandbox.RuntimeHandler]
+	} else {
+		ociRuntime = c.config.DefaultRuntime
+	}
+	runtimeOpts, err := generateRuntimeOptions(ociRuntime, c.config)
```

Comment:
Nit: we can probably make a utility function for figuring out the `sandboxPlatform`. I think I've seen this code block in a couple of more places.

Reply:
I was intending this new function to fill that role. Are you thinking of something else specifically?

Reply:
ugh lol, I cannot read apparently 🤦